tests/unit/test_schemas.py

```diff
@@ -15,35 +15,64 @@
 
 
 @pytest.fixture
-def schema_and_store():
+def tests_schema():
     # TODO: tmt package shall provide a helper function
     # for "load schemas for test/plan/story"
     schema_file = os.path.join(SCHEMADIR, 'tests.yaml')
-    schema = YAML(typ="safe").load(open(schema_file, encoding="utf-8"))
+    return YAML(typ="safe").load(open(schema_file, encoding="utf-8"))
 
-    schema_store = {}
+
+@pytest.fixture
+def stories_schema():
+    # TODO: tmt package shall provide a helper function
+    # for "load schemas for test/plan/story"
+    schema_file = os.path.join(SCHEMADIR, 'stories.yaml')
```

Comment:
Let's rename to `story.yaml` as @happz proposed.

Reply:
renamed in next update